precompiles/xvm/evm_sdk/flipper.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity ^0.8.0;
+
+interface XVM {
+    function xvm_call(
+        bytes calldata context,
+        bytes calldata to,
+        bytes calldata input,
+    ) external;
+}
+
+library Flipper {
+    const XVM XVM_PRECOMPILE = XVM(0x0000000000000000000000000000000000005005);
```

Comment:
```suggestion
    const XVM XVM_PRECOMPILE = XVM(0x5005);
```

Reply:
Chatting with @akorchyn , this is outdated and doesn't work.  Should fix before merging...

Reply:
It would be fixed in a separate repository.

Reply:
It would be fixed in a separate repository.

Reply:
It would be fixed in a separate repository.